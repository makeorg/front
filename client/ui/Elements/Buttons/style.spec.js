import React from 'react';
import renderer from 'react-test-renderer';
import {
  ButtonsWrapperStyle,
  UnstyledButtonStyle,
  ActiveButtonStyle,
  RedButtonStyle,
  GreyButtonStyle,
  SmallRedButtonStyle,
  SmallGreyButtonStyle,
  IconWrapperStyle,
  FacebookButtonStyle,
  GoogleButtonStyle,
  EmailButtonStyle,
  RedLinkButtonStyle,
  ButtonSmallWrapperStyle,
  CloseButtonStyle,
  QualifyButtonStyle,
  VoteIconStyle,
  VoteButtonStyle,
} from './style';

describe('ButtonElements/Styled', () => {
  describe('ButtonsWrapperStyle', () => {
    it('snapshot by default', () => {
      const component = renderer.create(<ButtonsWrapperStyle />).toJSON();
      expect(component).toMatchSnapshot();
    });
  });

  describe('UnstyledButtonStyle', () => {
    it('snapshot by default', () => {
      const component = renderer.create(<UnstyledButtonStyle />).toJSON();
      expect(component).toMatchSnapshot();
    });
  });

  describe('ActiveButtonStyle', () => {
    it('snapshot by default', () => {
      const component = renderer.create(<ActiveButtonStyle />).toJSON();
      expect(component).toMatchSnapshot();
    });
  });

  describe('RedButtonStyle', () => {
    it('snapshot by default', () => {
      const component = renderer.create(<RedButtonStyle />).toJSON();
      expect(component).toMatchSnapshot();
    });
  });

  describe('GreyButtonStyle', () => {
    it('snapshot by default', () => {
      const component = renderer.create(<GreyButtonStyle />).toJSON();
      expect(component).toMatchSnapshot();
    });
  });

  describe('SmallRedButtonStyle', () => {
    it('snapshot by default', () => {
      const component = renderer.create(<SmallRedButtonStyle />).toJSON();
      expect(component).toMatchSnapshot();
    });
  });

  describe('SmallGreyButtonStyle', () => {
    it('snapshot by default', () => {
      const component = renderer.create(<SmallGreyButtonStyle />).toJSON();
      expect(component).toMatchSnapshot();
    });
  });

  describe('IconWrapperStyle', () => {
    it('snapshot by default', () => {
      const component = renderer.create(<IconWrapperStyle />).toJSON();
      expect(component).toMatchSnapshot();
    });
  });

  describe('FacebookButtonStyle', () => {
    it('snapshot by default', () => {
      const component = renderer.create(<FacebookButtonStyle />).toJSON();
      expect(component).toMatchSnapshot();
    });
  });

  describe('GoogleButtonStyle', () => {
    it('snapshot by default', () => {
      const component = renderer.create(<GoogleButtonStyle />).toJSON();
      expect(component).toMatchSnapshot();
    });
  });

  describe('EmailButtonStyle', () => {
    it('snapshot by default', () => {
      const component = renderer.create(<EmailButtonStyle />).toJSON();
      expect(component).toMatchSnapshot();
    });
  });

  describe('RedLinkButtonStyle', () => {
    it('snapshot by default', () => {
      const component = renderer.create(<RedLinkButtonStyle />).toJSON();
      expect(component).toMatchSnapshot();
    });
  });

  describe('ButtonSmallWrapperStyle', () => {
    it('snapshot by default', () => {
      const component = renderer.create(<ButtonSmallWrapperStyle />).toJSON();
      expect(component).toMatchSnapshot();
    });
  });

  describe('CloseButtonStyle', () => {
    it('snapshot by default', () => {
      const component = renderer.create(<CloseButtonStyle />).toJSON();
      expect(component).toMatchSnapshot();
    });
  });

  describe('QualifyButtonStyle', () => {
    it('snapshot by default', () => {
      const component = renderer.create(<QualifyButtonStyle />).toJSON();
      expect(component).toMatchSnapshot();
    });
  });

  describe('VoteButtonStyle', () => {
    it('snapshot by default', () => {
      const component = renderer.create(<VoteButtonStyle />).toJSON();
      expect(component).toMatchSnapshot();
    });
  });

  describe('VoteIconStyle', () => {
    it('snapshot by default', () => {
      const component = renderer.create(<VoteIconStyle />).toJSON();
      expect(component).toMatchSnapshot();
    });
  });
});
