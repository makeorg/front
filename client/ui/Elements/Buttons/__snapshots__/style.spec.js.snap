// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonElements/Styled ActiveButtonStyle snapshot by default 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: baseline;
  -webkit-box-align: baseline;
  -ms-flex-align: baseline;
  align-items: baseline;
  font-family: 'Trade Gothic Bold Condensed',Arial,sans-serif;
  font-size: 14px;
  line-height: 1;
  border: none;
  border-radius: 20px;
  text-transform: uppercase;
  padding: 12px 25px 8px;
  color: rgb(255,255,255);
  background-color: rgb(222,26,66);
}

.c0 svg,
.c0 .tofill {
  fill: rgb(255,255,255);
}

.c0:disabled {
  color: rgb(74,74,74);
  background-color: rgb(229,229,229);
}

.c0:disabled svg,
.c0:disabled .tofill {
  fill: rgb(74,74,74);
}

@media (min-width:767px) {
  .c0 {
    font-size: 16px;
    padding: 13px 25px 10px;
  }
}

@media (min-width:1140px) {
  .c0 {
    font-size: 18px;
  }
}

<button
  className="sc-11o2adg-2 c0"
/>
`;

exports[`ButtonElements/Styled ButtonSmallWrapperStyle snapshot by default 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-flow: column;
  -ms-flex-flow: column;
  flex-flow: column;
  width: 100%;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  margin: 10px 0;
  max-width: 410px;
}

@media (min-width:767px) {
  .c0 {
    -webkit-flex-flow: row;
    -ms-flex-flow: row;
    flex-flow: row;
    margin: 20px 0;
  }
}

<div
  className="sc-11o2adg-0 c0"
/>
`;

exports[`ButtonElements/Styled ButtonsWrapperStyle snapshot by default 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-flow: column;
  -ms-flex-flow: column;
  flex-flow: column;
  width: 100%;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  margin: 10px 0;
}

@media (min-width:767px) {
  .c0 {
    -webkit-flex-flow: row;
    -ms-flex-flow: row;
    flex-flow: row;
    margin: 20px 0;
  }
}

<div
  className="c0"
/>
`;

exports[`ButtonElements/Styled CloseButtonStyle snapshot by default 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  padding: 0;
  border: none;
  background-color: transparent;
  position: absolute;
  top: 20px;
  right: 20px;
  fill: rgb(222,26,66);
  z-index: 1;
  font-size: 16px;
}

@media (min-width:767px) {
  .c0 {
    top: 30px;
    right: 30px;
  }
}

<button
  className="sc-11o2adg-1 c0"
/>
`;

exports[`ButtonElements/Styled EmailButtonStyle snapshot by default 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: baseline;
  -webkit-box-align: baseline;
  -ms-flex-align: baseline;
  align-items: baseline;
  font-family: 'Trade Gothic Bold Condensed',Arial,sans-serif;
  font-size: 14px;
  line-height: 1;
  border: none;
  border-radius: 20px;
  text-transform: uppercase;
  padding: 12px 25px 8px;
  color: rgb(255,255,255);
  background-color: rgb(222,26,66);
  width: 100%;
  margin: 0 5px;
  padding: 5px 15px;
  color: rgb(255,255,255);
  margin-top: 10px;
  background-color: rgb(222,26,66);
}

.c0 svg,
.c0 .tofill {
  fill: rgb(255,255,255);
}

.c0:disabled {
  color: rgb(74,74,74);
  background-color: rgb(229,229,229);
}

.c0:disabled svg,
.c0:disabled .tofill {
  fill: rgb(74,74,74);
}

.c0 svg {
  fill: rgb(255,255,255);
}

@media (min-width:767px) {
  .c0 {
    font-size: 16px;
    padding: 13px 25px 10px;
  }
}

@media (min-width:1140px) {
  .c0 {
    font-size: 18px;
  }
}

@media (min-width:767px) {
  .c0 {
    padding: 10px 20px;
  }
}

@media (min-width:767px) {
  .c0 {
    margin-top: 0;
  }
}

<button
  className="sc-11o2adg-2 sc-11o2adg-7 sc-11o2adg-9 c0"
/>
`;

exports[`ButtonElements/Styled FacebookButtonStyle snapshot by default 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: baseline;
  -webkit-box-align: baseline;
  -ms-flex-align: baseline;
  align-items: baseline;
  font-family: 'Trade Gothic Bold Condensed',Arial,sans-serif;
  font-size: 14px;
  line-height: 1;
  border: none;
  border-radius: 20px;
  text-transform: uppercase;
  padding: 12px 25px 8px;
  color: rgb(255,255,255);
  background-color: rgb(222,26,66);
  width: 100%;
  margin: 0 5px;
  padding: 5px 15px;
  color: rgb(255,255,255);
  background-color: rgb(58,89,152);
}

.c0 svg,
.c0 .tofill {
  fill: rgb(255,255,255);
}

.c0:disabled {
  color: rgb(74,74,74);
  background-color: rgb(229,229,229);
}

.c0:disabled svg,
.c0:disabled .tofill {
  fill: rgb(74,74,74);
}

.c0 svg {
  fill: rgb(255,255,255);
}

@media (min-width:767px) {
  .c0 {
    font-size: 16px;
    padding: 13px 25px 10px;
  }
}

@media (min-width:1140px) {
  .c0 {
    font-size: 18px;
  }
}

@media (min-width:767px) {
  .c0 {
    padding: 10px 20px;
  }
}

<button
  className="sc-11o2adg-2 sc-11o2adg-7 sc-11o2adg-9 c0"
/>
`;

exports[`ButtonElements/Styled GoogleButtonStyle snapshot by default 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: baseline;
  -webkit-box-align: baseline;
  -ms-flex-align: baseline;
  align-items: baseline;
  font-family: 'Trade Gothic Bold Condensed',Arial,sans-serif;
  font-size: 14px;
  line-height: 1;
  border: none;
  border-radius: 20px;
  text-transform: uppercase;
  padding: 12px 25px 8px;
  color: rgb(255,255,255);
  background-color: rgb(222,26,66);
  width: 100%;
  margin: 0 5px;
  padding: 5px 15px;
  color: rgb(255,255,255);
  margin-top: 10px;
  background-color: rgb(216,58,44);
}

.c0 svg,
.c0 .tofill {
  fill: rgb(255,255,255);
}

.c0:disabled {
  color: rgb(74,74,74);
  background-color: rgb(229,229,229);
}

.c0:disabled svg,
.c0:disabled .tofill {
  fill: rgb(74,74,74);
}

.c0 svg {
  fill: rgb(255,255,255);
}

@media (min-width:767px) {
  .c0 {
    font-size: 16px;
    padding: 13px 25px 10px;
  }
}

@media (min-width:1140px) {
  .c0 {
    font-size: 18px;
  }
}

@media (min-width:767px) {
  .c0 {
    padding: 10px 20px;
  }
}

@media (min-width:767px) {
  .c0 {
    margin-top: 0;
  }
}

<button
  className="sc-11o2adg-2 sc-11o2adg-7 sc-11o2adg-9 c0"
/>
`;

exports[`ButtonElements/Styled GreyButtonStyle snapshot by default 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: baseline;
  -webkit-box-align: baseline;
  -ms-flex-align: baseline;
  align-items: baseline;
  font-family: 'Trade Gothic Bold Condensed',Arial,sans-serif;
  font-size: 14px;
  line-height: 1;
  border: none;
  border-radius: 20px;
  text-transform: uppercase;
  padding: 12px 25px 8px;
  color: rgb(74,74,74);
  background-color: rgb(229,229,229);
}

.c0 svg {
  fill: rgb(222,26,66);
}

.c0:hover,
.c0:focus {
  color: rgb(74,74,74);
}

@media (min-width:767px) {
  .c0 {
    font-size: 16px;
    padding: 13px 25px 10px;
  }
}

@media (min-width:1140px) {
  .c0 {
    font-size: 18px;
  }
}

<button
  className="sc-11o2adg-2 c0"
/>
`;

exports[`ButtonElements/Styled IconWrapperStyle snapshot by default 1`] = `
.c0 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-box-pack: start;
  -webkit-justify-content: flex-start;
  -ms-flex-pack: start;
  justify-content: flex-start;
  -webkit-align-content: center;
  -ms-flex-line-pack: center;
  align-content: center;
  font-size: 12px;
  margin-right: 5px;
}

.c0 svg {
  width: 12px;
  height: 12px;
}

@media (min-width:1140px) {
  .c0 svg {
    width: 14px;
    height: 14px;
  }
}

<span
  className="c0"
/>
`;

exports[`ButtonElements/Styled QualifyButtonStyle snapshot by default 1`] = `
.c0 {
  font-family: 'Circular Standard Bold',Arial,sans-serif;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  width: 100%;
  border-width: 2px;
  font-size: 11.2px;
  line-height: 20px;
  border-style: solid;
  padding: 0 10px;
  border-radius: 36px;
  background-color: rgb(255,255,255);
}

.c0.qualified {
  color: rgb(255,255,255);
}

@media (min-width:360px) {
  .c0 {
    font-size: 12.6px;
    line-height: 23px;
  }
}

@media (min-width:1140px) {
  .c0 {
    font-size: 14px;
    line-height: 26px;
  }
}

<button
  className="c0"
/>
`;

exports[`ButtonElements/Styled RedButtonStyle snapshot by default 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: baseline;
  -webkit-box-align: baseline;
  -ms-flex-align: baseline;
  align-items: baseline;
  font-family: 'Trade Gothic Bold Condensed',Arial,sans-serif;
  font-size: 14px;
  line-height: 1;
  border: none;
  border-radius: 20px;
  text-transform: uppercase;
  padding: 12px 25px 8px;
  color: rgb(255,255,255);
  background-color: rgb(222,26,66);
}

.c0 svg {
  fill: rgb(255,255,255);
}

@media (min-width:767px) {
  .c0 {
    font-size: 16px;
    padding: 13px 25px 10px;
  }
}

@media (min-width:1140px) {
  .c0 {
    font-size: 18px;
  }
}

<button
  className="sc-11o2adg-2 c0"
/>
`;

exports[`ButtonElements/Styled RedLinkButtonStyle snapshot by default 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  padding: 0;
  border: none;
  background-color: transparent;
  display: inline-block;
  color: rgb(222,26,66);
  -webkit-text-decoration: underline;
  text-decoration: underline;
  margin: 0 5px;
}

<button
  className="sc-11o2adg-1 c0"
/>
`;

exports[`ButtonElements/Styled SmallGreyButtonStyle snapshot by default 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: baseline;
  -webkit-box-align: baseline;
  -ms-flex-align: baseline;
  align-items: baseline;
  font-family: 'Trade Gothic Bold Condensed',Arial,sans-serif;
  font-size: 14px;
  line-height: 1;
  border: none;
  border-radius: 20px;
  text-transform: uppercase;
  padding: 12px 25px 8px;
  color: rgb(74,74,74);
  background-color: rgb(229,229,229);
  padding: 5px 15px 2.5px;
}

.c0 svg {
  fill: rgb(222,26,66);
}

.c0:hover,
.c0:focus {
  color: rgb(74,74,74);
}

@media (min-width:767px) {
  .c0 {
    font-size: 16px;
    padding: 13px 25px 10px;
  }
}

@media (min-width:1140px) {
  .c0 {
    font-size: 18px;
  }
}

@media (min-width:767px) {
  .c0 {
    padding: 10px 20px;
  }
}

<button
  className="sc-11o2adg-2 sc-11o2adg-4 c0"
/>
`;

exports[`ButtonElements/Styled SmallRedButtonStyle snapshot by default 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: baseline;
  -webkit-box-align: baseline;
  -ms-flex-align: baseline;
  align-items: baseline;
  font-family: 'Trade Gothic Bold Condensed',Arial,sans-serif;
  font-size: 14px;
  line-height: 1;
  border: none;
  border-radius: 20px;
  text-transform: uppercase;
  padding: 12px 25px 8px;
  color: rgb(255,255,255);
  background-color: rgb(222,26,66);
  padding: 5px 15px 2.5px;
}

.c0 svg {
  fill: rgb(255,255,255);
}

@media (min-width:767px) {
  .c0 {
    font-size: 16px;
    padding: 13px 25px 10px;
  }
}

@media (min-width:1140px) {
  .c0 {
    font-size: 18px;
  }
}

@media (min-width:767px) {
  .c0 {
    padding: 10px 20px;
  }
}

<button
  className="sc-11o2adg-2 sc-11o2adg-3 c0"
/>
`;

exports[`ButtonElements/Styled UnstyledButtonStyle snapshot by default 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  padding: 0;
  border: none;
  background-color: transparent;
}

<button
  className="c0"
/>
`;

exports[`ButtonElements/Styled VoteButtonStyle snapshot by default 1`] = `
.c0 {
  position: relative;
  z-index: 0;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  width: 38px;
  height: 38px;
  min-width: 38px;
  min-height: 38px;
  border-width: 2px;
  border-style: solid;
  border-radius: 50%;
  overflow: hidden;
  padding: 0;
  background-color: rgb(255,255,255);
  -webkit-transform: scale(1);
  -ms-transform: scale(1);
  transform: scale(1);
  -webkit-transition: -webkit-transform 0.1s ease-in;
  -webkit-transition: transform 0.1s ease-in;
  transition: transform 0.1s ease-in;
}

.c0:hover,
.c0:focus {
  -webkit-transform: scale(1.1);
  -ms-transform: scale(1.1);
  transform: scale(1.1);
}

.c0.agree {
  color: rgb(80,122,31);
  border-color: rgb(80,122,31);
}

.c0.disagree {
  color: rgb(190,24,44);
  border-color: rgb(190,24,44);
}

.c0.neutral {
  color: rgb(117,117,117);
  border-color: rgb(117,117,117);
}

.c0.animated {
  box-shadow: 0 0 0 0;
  -webkit-animation: hnlfsR 0.5s 1;
  animation: hnlfsR 0.5s 1;
  -webkit-transform: scale(0.9);
  -ms-transform: scale(0.9);
  transform: scale(0.9);
}

.c0.animated.disagree {
  -webkit-animation: MCVOt 0.5s 1;
  animation: MCVOt 0.5s 1;
}

.c0.agree.voted {
  background-color: rgb(80,122,31);
}

.c0.disagree.voted {
  background-color: rgb(190,24,44);
}

.c0.neutral.voted {
  background-color: rgb(117,117,117);
}

.c0.voted {
  color: rgb(255,255,255);
}

@media (min-width:360px) {
  .c0 {
    width: 43px;
    height: 43px;
    min-width: 43px;
    min-height: 43px;
  }
}

@media (min-width:1140px) {
  .c0 {
    width: 48px;
    height: 48px;
    min-width: 48px;
    min-height: 48px;
  }
}

<button
  className="c0"
/>
`;

exports[`ButtonElements/Styled VoteIconStyle snapshot by default 1`] = `
.c0 {
  position: absolute;
  top: 50%;
  left: 50%;
  z-index: 0;
  width: 16px;
}

.c0.agree {
  -webkit-transform: translate(-50%,-50%);
  -ms-transform: translate(-50%,-50%);
  transform: translate(-50%,-50%);
}

.c0.disagree {
  -webkit-transform: translate(-50%,-50%) rotate(180deg) scaleX(-1);
  -ms-transform: translate(-50%,-50%) rotate(180deg) scaleX(-1);
  transform: translate(-50%,-50%) rotate(180deg) scaleX(-1);
}

.c0.neutral {
  -webkit-transform: translate(-50%,-50%) rotate(-90deg);
  -ms-transform: translate(-50%,-50%) rotate(-90deg);
  transform: translate(-50%,-50%) rotate(-90deg);
}

.c0.agree .tofill {
  fill: rgb(80,122,31);
}

.c0.disagree .tofill {
  fill: rgb(190,24,44);
}

.c0.neutral .tofill {
  fill: rgb(117,117,117);
}

.c0.voted .tofill {
  fill: rgb(255,255,255);
}

@media (min-width:360px) {
  .c0 {
    width: 22px;
  }
}

@media (min-width:1140px) {
  .c0 {
    width: 25px;
  }
}

<svg
  className="c0"
  height={25}
  viewBox="0 0 25 25"
  width={25}
>
  <g
    fill="none"
    fillRule="evenodd"
  >
    <path
      className="tofill"
      d="M5.388 20.031c.17-.17.255-.37.255-.602a.824.824 0 00-.255-.603.824.824 0 00-.602-.255.824.824 0 00-.603.255.824.824 0 00-.254.603.82.82 0 00.254.602c.17.17.37.255.603.255a.824.824 0 00.602-.255zm15.683-8.317c0-.455-.174-.855-.522-1.198-.348-.344-.745-.516-1.192-.516h-4.714c0-.518.214-1.23.643-2.136.428-.906.643-1.623.643-2.15 0-.875-.143-1.522-.429-1.942-.286-.42-.857-.63-1.714-.63-.232.233-.402.612-.51 1.14a24.555 24.555 0 01-.408 1.68c-.165.594-.43 1.083-.797 1.467-.196.205-.54.611-1.03 1.218-.037.045-.14.179-.309.402a26.028 26.028 0 01-.884 1.118 7.61 7.61 0 01-.536.59c-.16.156-.332.314-.515.475a2.31 2.31 0 01-.536.362 1.15 1.15 0 01-.475.12h-.429v8.572h.429c.116 0 .256.013.422.04.165.027.312.056.442.087.129.031.299.08.508.147a20.982 20.982 0 01.945.322c.214.076.343.122.388.14 1.884.652 3.41.978 4.58.978h1.621c1.714 0 2.571-.746 2.571-2.237 0-.232-.022-.482-.067-.75.268-.142.48-.377.637-.703a2.26 2.26 0 00.234-.984c0-.33-.08-.638-.241-.924.473-.447.71-.978.71-1.594 0-.223-.045-.47-.134-.743-.09-.273-.201-.485-.335-.636.286-.01.525-.22.716-.63.192-.41.288-.772.288-1.085zm1.715-.013c0 .795-.219 1.522-.657 2.183.08.295.121.603.121.924 0 .688-.17 1.33-.509 1.929.027.187.04.38.04.575 0 .902-.268 1.697-.803 2.384.009 1.242-.37 2.221-1.139 2.94-.768.719-1.781 1.078-3.04 1.078h-1.728c-.857 0-1.703-.1-2.538-.301-.834-.2-1.8-.493-2.9-.877C8.599 22.179 7.984 22 7.787 22H3.929c-.474 0-.878-.167-1.212-.502a1.652 1.652 0 01-.503-1.212v-8.572c0-.473.168-.877.503-1.212A1.652 1.652 0 013.929 10h3.67c.32-.214.932-.906 1.834-2.076a25.8 25.8 0 011.433-1.714c.214-.223.373-.605.476-1.145.102-.54.238-1.105.408-1.694.17-.59.446-1.072.83-1.447a1.7 1.7 0 011.206-.495c.75 0 1.424.145 2.022.435.598.29 1.054.743 1.366 1.36.313.615.469 1.446.469 2.49 0 .83-.214 1.688-.643 2.572h2.357c.929 0 1.732.339 2.41 1.018.68.678 1.019 1.477 1.019 2.397z"
      fill="#fff"
    />
    <path
      d="M0 0h25v25H0z"
    />
  </g>
</svg>
`;
